.github/workflows/demo.yml

```diff
@@ -1,46 +1,37 @@
 on: [ push, pull_request ]
 
 jobs:
-    deployment_keys_demo:
-        strategy:
-            fail-fast: false
-            matrix:
-                os: [ ubuntu-latest, macOS-latest, windows-latest ]
-        runs-on: ${{ matrix.os }}
-        steps:
-            -   uses: actions/checkout@v3
-            -   name: Setup key
-                uses: ./
-                with:
-                    ssh-private-key: |
-                        ${{ secrets.MPDUDE_TEST_1_DEPLOY_KEY }}
-                        ${{ secrets.MPDUDE_TEST_2_DEPLOY_KEY }}
-            -   run: |
-                    git clone https://github.com/mpdude/test-1.git test-1-http
-                    git clone git@github.com:mpdude/test-1.git test-1-git
-                    git clone ssh://git@github.com/mpdude/test-1.git test-1-git-ssh
-                    git clone https://github.com/mpdude/test-2.git test-2-http
-                    git clone git@github.com:mpdude/test-2.git test-2-git
-                    git clone ssh://git@github.com/mpdude/test-2.git test-2-git-ssh
-
-    docker_demo:
-        runs-on: ubuntu-latest
-        container:
-            image: ubuntu:latest
-        steps:
-            -   uses: actions/checkout@v3
-            -   run: apt update && apt install -y openssh-client git
-            -   name: Setup key
-                uses: ./
-                with:
-                    ssh-private-key: |
-                        ${{ secrets.MPDUDE_TEST_1_DEPLOY_KEY }}
-                        ${{ secrets.MPDUDE_TEST_2_DEPLOY_KEY }}
-            -   run: |
-                    git clone https://github.com/mpdude/test-1.git test-1-http
-                    git clone git@github.com:mpdude/test-1.git test-1-git
-                    git clone ssh://git@github.com/mpdude/test-1.git test-1-git-ssh
-                    git clone https://github.com/mpdude/test-2.git test-2-http
-                    git clone git@github.com:mpdude/test-2.git test-2-git
-                    git clone ssh://git@github.com/mpdude/test-2.git test-2-git-ssh
-
+  vmo_deployment_keys_demo:
+    runs-on: ubuntu-latest
+    container:
+      image: ubuntu:latest
+    steps:
+      -   uses: actions/checkout@v3
+      -   run: apt update && apt install -y openssh-client git curl
+      -   if: ${{ env.ACT }}
+          name: Hack container for local development
+          run: |
+            curl -fsSL https://deb.nodesource.com/setup_16.x | sudo -E bash -
+            apt-get install -y nodejs
```

Comment:
What is this for, given the next step is "Setup node"? Does that next step not work locally and becomes a "no-op" if nodejs is already installed and at the right version?

Reply:
this is "hack" for local container when using with `act` tool ( only executed when `if: ${{ env.ACT }}`) - got same error as others with node missing. Some "known" issue with the tool when using following combo:
```
    runs-on: ubuntu-latest
    container:
      image: ubuntu:latest
```

This is error without this:
<img width="1207" alt="image" src="https://user-images.githubusercontent.com/3975860/198269191-573768f9-03dc-4b78-b0d1-df4aee3556eb.png">

Reply:
Aha... so the action to setup node uses node to run :) - how nice.
But yeah, I noticed in an open source project i contributed to that Github's image "ubuntu-latest" is actually different compared to the "ubuntu:latest" and has more things installed already.

Btw, why do you use (overwrite) "ubuntu:latest" as the image and not let `act` pick the right image for "ubuntu-latest"? `act` has an image `ghcr.io/catthehacker/ubuntu:act-latest` - maybe that one contains `node` already like github's runner image?

Reply:
It seems their image does install node: https://github.com/catthehacker/docker_images/blob/master/linux/ubuntu/scripts/act.sh#L128

Reply:
this was taken from parent fork, to test as container. will see if I have possibility to dig more next week